bls/algebra/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "bls_zexe"
```

Comment:
Nitpick - isn't it conventional to be hyphen like it was?

Reply:
Changed!